<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttendanceRecord</class>
 <widget class="QWidget" name="AttendanceRecord">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1360</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>考勤记录</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="staffIdLabel">
       <property name="text">
        <string>员工ID：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="staffIdValueLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="attendanceDateLabel">
       <property name="text">
        <string>考勤日期：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="attendanceDateValueLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="checkInTimeLabel">
       <property name="text">
        <string>上班打卡时间：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="checkInTimeValueLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="checkOutTimeLabel">
       <property name="text">
        <string>下班打卡时间：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="checkOutTimeValueLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="checkInButton">
     <property name="text">
      <string>上班打卡</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="checkOutButton">
     <property name="text">
      <string>下班打卡</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
